### PR TITLE
Remove some ex-stanford folk

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

**Review by mjgiarlo — APPROVED** [review]

File: script/grant_revoke_gem_authority.rb [hunk 1]
```diff
@@ -56,7 +54,6 @@
   'mkorcy' => "mkorcy@gmail.com",
   'ndushay' => "ndushay@stanford.edu",
   'tpendragon' => "tpendragon@princeton.edu",
-  'carrickr' => 'carrickr@umich.edu',
   'no_reply' => 'johnson.tom@gmail.com',
   'revgum' => 'revgum@gmail.com',
```
Comment (mjgiarlo) [inline script/grant_revoke_gem_authority.rb:58]: I believe Josh Gum has left OSU.

File: script/grant_revoke_gem_authority.rb [hunk 2]
```diff
@@ -25,15 +25,13 @@
 KNOWN_COMMITTER_EMAIL_ADDRESSES = {
   'aaron-collier' => 'aaron.collier@stanford.edu',
   'awead' => "awead@users.noreply.github.com",
-  'atz' => 'ohiocore@gmail.com',
   'barmintor' => "armintor@gmail.com",
   'bess' => "bess@curationexperts.com",
   'cam156' => "cam156@psu.edu",
```
Comment (mjgiarlo) [inline script/grant_revoke_gem_authority.rb:30]: I believe Carolyn Cole has left Penn State.

File: script/grant_revoke_gem_authority.rb (hunk 1 above)
Comment (wickr) [reply script/grant_revoke_gem_authority.rb:58]: Josh is still at OSU but not at the Library so can be removed.

Comment (revgum) [reply script/grant_revoke_gem_authority.rb:58]: Gracias!

**Comment by cjcolvar** [comment]
I merged this as is and am opening a new one with the suggested change.

**Pull request merged**